WordPressKit/WordPressKit/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -102,6 +104,39 @@ - (void)createTag:(RemotePostTag *)tag
                          } failure:failure];
 }
 
+- (void)updateTag:(RemotePostTag *)tag
+          success:(nullable void (^)(RemotePostTag *tag))success
+          failure:(nullable void (^)(NSError *error))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionary];
+    [extraParameters setObject:tag.tagID ?: [NSNull null] forKey:TaxonomyXMLRPCIDParameter];
+    [extraParameters setObject:tag.name ?: [NSNull null] forKey:TaxonomyXMLRPCNameParameter];
+    [extraParameters setObject:tag.description ?: [NSNull null] forKey:TaxonomyXMLRPCDescriptionParameter];
```

Comment:
Should `tag.description` be `tag.tagDescription`?   When I inspect the parameters being passed I see something like `"<RemotePostTag: 0x60400065b750> Tag6[14]";` for its value.